webapi/src/it/scala/org/knora/webapi/it/v2/KnoraSipiIntegrationV2ITSpec.scala

```diff
@@ -429,6 +432,67 @@ class KnoraSipiIntegrationV2ITSpec
       assert(savedImage.height == marblesHeight)
     }
 
+    "create a resource with a still image file without processing" in {
+      // Upload the image to Sipi.
+      val sipiUploadResponse: SipiUploadWithoutProcessingResponse =
+        uploadWithoutProcessingToSipi(
+          loginToken = loginToken,
+          filesToUpload = Seq(FileToUpload(path = pathToJp2, mimeType = org.apache.http.entity.ContentType.IMAGE_JPEG))
+        )
+
+      val uploadedFile: SipiUploadWithoutProcessingResponseEntry = sipiUploadResponse.uploadedFiles.head
+      uploadedFile.filename should ===(jp2OriginalFilename)
+
+      // Create the resource in the API.
+
+      val jsonLdEntity = UploadFileRequest
+        .make(
+          fileType = FileType.StillImageFile(),
+          internalFilename = uploadedFile.filename
+        )
+        .toJsonLd(
+          className = Some("ThingPicture"),
+          ontologyName = "anything"
+        )
+
+      val request = Post(
+        s"$baseApiUrl/v2/resources",
+        HttpEntity(RdfMediaTypes.`application/ld+json`, jsonLdEntity)
+      ) ~> addCredentials(BasicHttpCredentials(anythingUserEmail, password))
+      val responseJsonDoc: JsonLDDocument = getResponseJsonLD(request)
+      stillImageResourceIri.set(responseJsonDoc.body.requireIDAsKnoraDataIri.toString)
+
+      // Get the resource from Knora.
+      val knoraGetRequest          = Get(s"$baseApiUrl/v2/resources/${URLEncoder.encode(stillImageResourceIri.get, "UTF-8")}")
+      val resource: JsonLDDocument = getResponseJsonLD(knoraGetRequest)
+      assert(
+        resource.body.requireTypeAsKnoraApiV2ComplexTypeIri.toString == "http://0.0.0.0:3333/ontology/0001/anything/v2#ThingPicture"
+      )
+
+      // Get the new file value from the resource.
+
+      val savedValues: JsonLDArray = getValuesFromResource(
+        resource = resource,
+        propertyIriInResult = OntologyConstants.KnoraApiV2Complex.HasStillImageFileValue.toSmartIri
+      )
+
+      val savedValue: JsonLDValue = if (savedValues.value.size == 1) {
+        savedValues.value.head
+      } else {
+        throw AssertionException(s"Expected one file value, got ${savedValues.value.size}")
+      }
```

Comment:
I'm not sure if this makes sense at all.

Reply:
No, it probably doesn't make much sense. I copied this test from above and just replaced the route. I removed this extra check now.